src/test/resources/software/amazon/smithy/lsp/ext/models/empty.smithy

```diff
@@ -0,0 +1,8 @@
+$version: "1.0"
```

Comment:
Doesn't look like this file gets loaded anywhere in the new tests, is it needed? If so, I imagined it was going to be literally an empty Smithy model file given the name, so maybe rename it?

Reply:
The model file does get loaded here: [SmithyProject.java#L190](https://github.com/awslabs/smithy-language-server/blob/main/src/main/java/software/amazon/smithy/lsp/ext/SmithyProject.java#L190).

I updated the model file name in [97644ae](https://github.com/awslabs/smithy-language-server/pull/40/commits/97644ae91ea1e1f4d063eb7f4a814157f3d865c1) to make that less confusing.